dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/utils/McpClient.java

```diff
@@ -40,6 +41,7 @@
 public class McpClient extends ExternalResource {
 
     private static final String ACCEPT_HEADER = "application/json, text/event-stream";
+    private static final String CONTENT_TYPE = "application/json";
```

Comment:
```suggestion
    public static final String APPLICATION_JSON = "application/json";
```

We do often want to put hard-coded strings into constants, but it shouldn't obscure what's being tested.

When we later write this:
```
        assertTrue(contentType.contains(CONTENT_TYPE));
```
It's harder to see what's being tested.

In some cases it may make sense to call it `EXPECTED_CONTENT_TYPE` or something if the actual content type is complex.

I'd also be tempted to make it public so you can reference it from other tests which need the same value.